admin/apple-actions/index/class-export.php

```diff
@@ -57,7 +57,7 @@ public function fetch_exporter() {
 		$post       = get_post( $this->id );
 
 		// Build the excerpt if required
-		$excerpt = ( empty( $post->post_excerpt ) ) ? wp_trim_excerpt( $post->post_content ) : $post->post_excerpt;
+		$excerpt = ( empty( $post->post_excerpt ) ) ? wp_trim_words( strip_shortcodes( $post->post_content, 55 ) ) : $post->post_excerpt;
```

Comment:
`55` is a parameter on `wp_trim_words`, not `strip_shortcodes`.

Pull requests are no longer being accepted without unit tests if you can please add one for this.

Thanks.